pixi.toml

```diff
@@ -12,8 +12,6 @@ pre-commit = "*"
 git_root = "*"
 pip = "*"
 python = ">=3.10"
-setuptools-scm = "*"
-setuptools = ">=61"
```

Comment:
these are actually still needed for the `postinstall` task

Reply:
Ah, I see!